### PR TITLE
app-misc/mosquitto: bump to version 1.5.8

### PR DESCRIPTION
**Pull request by oxr463**
Package-Manager: Portage-2.3.62, Repoman-2.3.11
Signed-off-by: Lucas Ramage <ramage.lucas@protonmail.com>

### CODE REVIEW

File: app-misc/mosquitto/mosquitto-1.5.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	dodoc readme.md CONTRIBUTING.md ChangeLog.txt
+	doinitd "${FILESDIR}"/mosquitto
+	insinto /etc/mosquitto
+	doins mosquitto.conf
+	systemd_dounit "${FILESDIR}/mosquitto.service"
+
+	if use examples; then
+		docompress -x "/usr/share/doc/${PF}/examples"
+		docinto "/usr/share/doc/${PF}"
+		doins -r examples
```
Comment (mgorny) [inline app-misc/mosquitto/mosquitto-1.5.8.ebuild]: docinto doesn't combine with doins.

File: app-misc/mosquitto/mosquitto-1.5.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit systemd user toolchain-funcs python-any-r1
```
Comment (mgorny) [inline app-misc/mosquitto/mosquitto-1.5.8.ebuild]: Sort, please.

File: app-misc/mosquitto/mosquitto-1.5.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+SRC_URI="https://mosquitto.org/files/source/${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="bridge examples +persistence +srv ssl tcpd test websockets"
+
+REQUIRED_USE="test? ( bridge )"
+
+RDEPEND="tcpd? ( sys-apps/tcp-wrappers )
```
Comment (mgorny) [inline app-misc/mosquitto/mosquitto-1.5.8.ebuild]: Sort this too.

**Comment by oxr463** [comment]
@mgorny, please check my fixes.

File: app-misc/mosquitto/mosquitto-1.5.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	fowners mosquitto:mosquitto /var/lib/mosquitto
+	dodoc readme.md CONTRIBUTING.md ChangeLog.txt
+	doinitd "${FILESDIR}"/mosquitto
+	insinto /etc/mosquitto
+	doins mosquitto.conf
+	systemd_dounit "${FILESDIR}/mosquitto.service"
+
+	if use examples; then
+		docompress -x "/usr/share/doc/${PF}/examples"
+		docinto "/usr/share/doc/${PF}"
```
Comment (mgorny) [inline app-misc/mosquitto/mosquitto-1.5.8.ebuild]: docinto isn't supposed to take full path, and you're trying to set it to the default anyway, which makes no sense. Just remove it.

Comment (mgorny) [reply app-misc/mosquitto/mosquitto-1.5.8.ebuild]: Hm, or I'll just do it for you while merging.

**Pull request closed without merging**